tap/source/tcp_packet_source.go

```diff
@@ -116,6 +117,10 @@ func (source *tcpPacketSource) close() {
 }
 
 func (source *tcpPacketSource) readPackets(ipdefrag bool, packets chan<- TcpPacketInfo) {
+	if os.Getenv("MIZU_TAPPER_NO_PCAP") == "true" {
```

Comment:
I'm not sure what's the exact purpose of this disabling but it basically nullifies the capture. So, I would not initialize the handle at all if we want to disable the capturing. It disables this `libpcap` FFI call https://github.com/google/gopacket/blob/v1.1.19/pcap/pcap_unix.go#L398

Reply:
It's a baseline for other measurements where everything is initialized but no work is done.
This is on purpose.